runtime/lua/vim/lsp/util.lua

```diff
@@ -781,34 +781,44 @@ function M.convert_input_to_markdown_lines(input, contents)
   return contents
 end
 
+--- Returns the line/column-based position in `contents` at the given offset.
+---
+---@param offset integer
+---@param contents string[]
+---@return Range2
```

Comment:
Coming from https://github.com/neovim/neovim/pull/30460#discussion_r1798438892, this seems to be fixed:

```suggestion
---@return Range2
+++@return {[1]: integer, [2]: integer} row, col
```

Because Range2 means (start row, start end), not a position.

note: since this is just a local helper function, we can also simply return two integers rather than a table.

Reply:
I'll keep the table so that I can still just use `list_extend` without introducing extra variables at the call sites.